src/main/java/seedu/duke/Duke.java

```diff
@@ -13,9 +14,9 @@ public Duke() {
      */
     public static void main(String[] args) {
         Formatter.printWelcomeMsg();
-
+        PlayerList.L1.add(new MediumSkill("Bruno"));
```

Comment:
Why is the player name Bruno, and the skill level medium? You should add a comment explaining why.